apps/api/internal/apperror/errorHandler.go

```
@@ -0,0 +1,40 @@
+	case NoTargetData, ReqBodyDecodeFailed, BadParam:
+		statusCode = http.StatusBadRequest
+	case RequiredAuthorizationHeader, CannotMakeValidator, Unauthorizated:
+		statusCode = http.StatusUnauthorized
+	case NotMatchUser:
+		statusCode = http.StatusForbidden
+	default:
+		statusCode = http.StatusInternalServerError
+	}
+
+	w.WriteHeader(statusCode)
+	json.NewEncoder(w).Encode(appErr)
```

golangci-lint (2.13.2)

[error] Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)